ipatests/test_webui/test_hostgroup.py

```diff
@@ -31,6 +31,15 @@
 import pytest
 
 
+def test_invalid_names(self, names, error_link):
```

Comment:
Test results show that this fail. The method has "test" in a name, which means that pytest picks it as something to run. IMO, the method needs to be renamed in order the test suite to succeed.